res/values/styles.xml

```diff
@@ -22,4 +22,20 @@
     <item name="android:layout_marginHorizontal">16dp</item>
     <item name="android:layout_gravity">center</item>
   </style>
+  <style name="appTheme" parent="@android:style/Theme.Material.Light">
+    <item name="android:textStyle">bold</item>
+    <item name="android:colorAccent">#ff2424ff</item>
+    <item name="android:colorBackground">#ffeeeeff</item>
```

Comment:
These colors don't look natural on my device. Why not use the default colors by not specifying them ?

Reply:
After meditating, I thought about the number of devices with different designs, and well, I thought about the topic more for new devices (Holo: I'm still alive), so I'm going to leave only the basic modifiers and leave the rest as default, for right, what api is your device?

Reply:
The point of my changes is to make the keyboard with more universal modifications, for different people, I love flat designs, but others don't, that's why I have to think more global and less local